### PR TITLE
ACR-1814: Add a new feature key for ABI info

### PR DESCRIPTION
**Pull request by dkson95**
1) platform.core.abi
2) platform.core.cpu.arch.riscv32
3) platform.core.cpu.arch.riscv64

### API Changes ###

 - ACR: ACR-1814 (new feature key)

### CODE REVIEW

File: docs/application/features/feature.md [hunk 1]
```diff
@@ -1869,6 +1869,18 @@ The version of Tizen Core API in the \[Major\].\[Minor\] format. For example, "1
 
 <div class="views-row views-row-119 views-row-odd">
 
+[http://tizen.org/feature/platform.core.abi](){#feature/platform.core.abi}
+------------------------------------------------------------------------------------
+
+**System information:**\
+The platform returns the ABI information (e.g. "arm_32", "arm_64", "riscv_32", "riscv_64", "x86", "x86_64) of a device.\
+
+[Edit Key]( "edit"){.edit}
```
Comment (dkson95) [inline docs/application/features/feature.md:1878]: I'm not sure on this @safir-srbd could you help ?^^

**Review by jh80chung — APPROVED** [review]
I'm not sure if someone needs to review. Anyway, looks good to me.

**Review by safir-srbd — COMMENTED** [review]
@dkson95 I have left a comment. Please check.

**Comment by dkson95** [comment]
> @dkson95 I have left a comment. Please check.

okay, updated via https://github.com/Samsung/tizen-docs/pull/1966/commits/384dfb46621c013819bed3cf1576e5b105d9d6e6
Thank you!

**Review by safir-srbd — APPROVED** [review]
@dkson95 LGTM. I am approving this PR for merging.

**Pull request merged**